x-pack/plugins/apm/public/components/app/service_map/Popover/backend_contents.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useContext, useEffect, useState } from 'react';
+import { ContentsProps } from '.';
+import { NodeStats } from '../../../../../common/service_map';
+import { useUrlParams } from '../../../../context/url_params_context/use_url_params';
+import { useApmRouter } from '../../../../hooks/use_apm_router';
+import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
+import { CytoscapeContext } from '../Cytoscape';
+import { StatsList } from './stats_list';
+
+export function BackendContents({ nodeData }: ContentsProps) {
+  const apmRouter = useApmRouter();
+  const {
+    urlParams: { environment, start, end },
+  } = useUrlParams();
+
+  const backendName = nodeData.label;
+
+  // We only want the backend stats for services upstream in the current view of
```

Comment:
why's that? do we do this for stats between instrumented services as well?

Reply:
It seemed like that made more sense. If I'm looking at the stats for a backend on the map I want to see those stats in terms of the communication between those services, not as a whole.

But you're right, we don't do this for instrumented services so I'll take it out.

Reply:
Updated in 549a910302c.